actions/ansible-docs-build-init/action.yml

```diff
@@ -43,12 +68,12 @@ runs:
             echo "::endgroup::"
 
             echo "::group::Initialize Sphinx"
-            antsibull-docs sphinx-init --use-current --dest-dir "${{ inputs.dest-dir }}" ${{ inputs.collections }}
+            antsibull-docs sphinx-init --use-current ${{ fromJSON(inputs.fail-on-error) && '--fail-on-error' || '' }} ${{ fromJSON(inputs.lenient) && '--lenient' || '' }} --dest-dir "${{ inputs.dest-dir }}" ${{ inputs.collections }}
             echo "::endgroup::"
         fi
 
         echo "::group::Install additional requirements"
-        pip install -r "${{ inputs.dest-dir }}/requirements.txt" --disable-pip-version-check
+        pip install -r "${{ inputs.dest-dir }}/requirements.txt"
```

Comment:
Why not keep `--disable-pip-version-check` (and add it to the above `pip install`s as well)?

Reply:
I set it an environment variable instead so it applies across the board and I don't have to keep repeating it

Reply:
Ah right. I noticed that when reviewing this earlier, but then forgot again about it when looking at it again...